recipe/meta.yaml

```
@@ -8,11 +8,11 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 33a5ef65df572669b74ee3ac6aafcbecee85f233d2f770b84743d5671a9e0fe0
+  sha256: 24bc8333046ae05bd4dbb00512525033df128f07f33876d43e5ecbcc4b7d06e1
 
 build:
   number: 0
-  skip: true  # [win or py<35]
+  skip: true  # [py<36]
```

```suggestion
  skip: true  # [win or py<36]
```
Sigh. I'm not sure why my windows tests work via gh actions, and pip wheel builds are fine, but this fails...but maybe the conda package should be linux and mac only!

---

It may have something to do with the compiler -- conda-forge uses the MS compiler by default. Sample failure is at https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=216081&view=logs&j=e5cdccbf-4751-5a24-7406-185c9d30d021&t=66e4a587-a723-53e9-873c-6c9a2c5971ea&l=700